src/Synapse/Synapse/Models/SynapseAnalyticsManagementClient.cs

```diff
@@ -1889,6 +1889,76 @@ public SqlDatabase CreateSqlDatabase(string resourceGroupName, string workspaceN
             }
         }
 
+        public virtual async Task<PSBackupModel> GetRecoverableSqlPool(string resourceGroupName, string workspaceName, string sqlPoolName)
+        {
+            try
+            {
+                var taskResponse = await _synapseManagementClient.WorkspaceManagedSqlServerRecoverableSqlpools.GetWithHttpMessagesAsync(resourceGroupName, workspaceName, sqlPoolName);
```

Comment:
>await _synapseManagementClient.WorkspaceManagedSqlServerRecoverableSqlpools.GetWithHttpMessagesAsync [](start = 35, length = 100)

`_synapseManagementClient.WorkspaceManagedSqlServerRecoverableSqlpools.Get` is more consistent with existing methods.